### PR TITLE
Add owner and status to .objects() and bump to v2.5.

### PR DESCRIPTION
**Pull request by mgoffin**
This adds the ability to search on owner and status using .objects().
Now that v2.5 is out we are also bumping to that.

### CODE REVIEW

File: pytx/pytx/common.py [hunk 1]
```diff
@@ -137,9 +137,9 @@ def to_dict(self):
 
     @classmethod
     def objects(cls, text=None, strict_text=False, type_=None, threat_type=None,
```
Comment (jessek) [inline pytx/pytx/common.py]: This is getting cluttered. Can you please put one argument per line?

File: pytx/pytx/request.py [hunk 2]
```diff
@@ -150,7 +150,7 @@ def validate_get(cls, limit, since, until):
     @classmethod
     def build_get_parameters(cls, text=None, strict_text=None, type_=None,
```
Comment (jessek) [inline pytx/pytx/request.py]: This is getting cluttered. Can you please put one argument per line?

**Comment by mgoffin** [comment]
Will do! I'll fix that up.

**Comment by mgoffin** [comment]
Yay for passing tests and easier to read function arguments :) Thanks for pointing that out @jessek. It's a heck of a lot more readable now!

**Pull request merged**